internal/lookout/ui/src/components/job-dialog/JobLogs.tsx

```diff
@@ -104,6 +105,13 @@ export default function JobLogs(props: JobLogsProps) {
           {props.error}
         </Alert>
       )}
+      <p>
```

Comment:
This will insert a <p> in all cases, probably you only want that block when there's an actual error

Reply:
Yes - good catch - fixed in commit 22f12d1c.